app/grants/migrations/0128_auto_20211122_2034.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.24 on 2021-11-22 20:34
+
+from django.db import migrations, models
+import grants.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grants', '0127_auto_20211029_0913'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='grantcollection',
+            options={'get_latest_by': 'created_on'},
+        ),
```

Comment:
This was also picked up when I ran make migrations. Let me know if I should cut it out.

Reply:
@thelostone-mc has fixed this in master 👍